06_gpu_and_ml/text-to-pokemon/text_to_pokemon/main.py

```diff
@@ -174,7 +174,11 @@ def diskcached_text_to_pokemon(prompt: str) -> list[bytes]:
 
 
 @stub.asgi(
```

Comment:
this needs to be updated to the newest web endpoint syntax – weird that the tests didn't cathc that

Reply:
Will do.